src/ducks/transfers/TransferPage.jsx

```diff
@@ -137,7 +152,12 @@ class TransferPage extends React.Component {
       amount: '',
       password: '',
       label: '',
-      date: new Date().toISOString().slice(0, 10)
+      date: new Date().toISOString().slice(0, 10),
+      showingPersonalInfo:
+        this.props.myself &&
+        this.props.myself.data &&
+        this.props.myself.data[0] &&
```

Comment:
Nit: `_.get(this.props, 'myself.data[0]')` makes this sort of code less error prone imo.